src/_includes/compass_teaser.html

```diff
@@ -1,4 +1,7 @@
-<section id="compass-by-codurance" class="compass-teaser">
+<section class="compass-teaser">
+  {%- assign anchor_id = "compass-by-codurance" -%}
```

Comment:
What is the use of the hyphen after the and before the %?

I've seen that for all the assign statements  for the anchor_id but I wasn't sure why they were needed.

Reply:
I added Jekyll Snippets to my VS Code, and they were put in automatically. 

I also wondered why they were being put in and when I looked it up it mentioned that it removes any generated white-space from the template.

Reply:
The scrolling works so it's not causing an issue in the code but just different to the documentation for liquid.